lib/catalog/submit_order_item.rb

```diff
@@ -21,7 +21,11 @@ def process
     rescue => e
       @order_item.mark_failed("Error Submitting Order Item: #{e.message}")
     ensure
-      order_item.update(:service_parameters => runtime_parameters)
+      begin
+        order_item.update(:service_parameters => runtime_parameters)
+      rescue => e
+        Rails.logger.error("Failed to update OrderItem #{order_item.id} with runtime parameters, error #{e.message}")
```

Comment:
Do you think this is also something we should be putting into the progress message for the users to see? Also if we're doing this for debugging purposes, should we log the `runtime_parameters` that caused the error?

Reply:
`runtime_parameters` was already called in the body, and it was caught and logged properly.
I now update the code to check `@runtime_paramters` only.